src/main/java/com/app/onlinebookstore/security/JwtAuthenticationFilter.java

```diff
@@ -18,7 +18,6 @@
 @Component
 @RequiredArgsConstructor
 public class JwtAuthenticationFilter extends OncePerRequestFilter {
-    private static final int START_TOKEN_INDEX = 7;
     private static final String BEARER_TOKEN = "Bearer ";
     private static final String AUTHORIZATION_TOKEN = "Authorization";

```

Comment:
Spring's HttpHeaders class already has this constant, you can consider using it instead